contracts/contracts/interfaces/IStrategy.sol

```diff
@@ -48,16 +48,10 @@ interface IStrategy {
     /**
      * @dev Collect reward tokens from the Strategy.
      */
-    function collectRewardToken() external;
+    function collectRewardTokens() external;
 
     /**
-     * @dev The address of the reward token for the Strategy.
+     * @dev The address array of the reward tokens for the Strategy.
      */
-    function rewardTokenAddress() external pure returns (address);
-
-    /**
-     * @dev The threshold (denominated in the reward token) over which the
-     * vault will auto harvest on allocate calls.
-     */
-    function rewardLiquidationThreshold() external pure returns (uint256);
+    function getRewardTokenAddresses() external pure returns (address[] memory);
```

Comment:
I don't think is is actually `pure`. It should be `view` since it reads from state.

Reply:
Yup true, thanks